zokrates_js/js/utils.js

```diff
@@ -0,0 +1,24 @@
+export function getAbsolutePath(basePath, relativePath) {
```

Comment:
can we have some tests for the absolute resolution?

Reply:
Added tests for absolute resoltion and stdlib resolving.